plugin-server/src/utils/db/hub.ts

```diff
@@ -107,6 +107,17 @@ export async function createHub(
             on the ca trust_cert for this purpose. */
             rejectUnauthorized: false,
         }
+    } else if (serverConfig.KAFKA_SECURITY_PROTOCOL === 'SSL' || serverConfig.KAFKA_SECURITY_PROTOCOL === 'SASL_SSL') {
```

Comment:
nit: can we set up an enum for these?

Reply:
Sure, would you also prefer an enum for the values in Python?

Reply:
Could be nice yeah, although I'm being more particular about TS as we use them across the board. Unsure how consistent we are about this on the Python side.